### ISSUE TITLE
Hi from version 1.1.1f to 1.1.1h we get SQL errors while importing matecat.sql

### ISSUE THREAD

**Issue by wsuphappyhour** [matecat/MateCat#599]
Hi from version 1.1.1f to 1.1.1h we get SQL errors while importing matecat.sql. Our MySQL version is 5.6.33, which one are you using?

Thanks.

**Comment by Ostico** [comment]
Hi @wsuphappyhour 
please refer to this comment.

https://github.com/matecat/MateCat/pull/597#issuecomment-272824467

**Comment by wsuphappyhour** [comment]
Hi,

Thanks, could you explain how to do those migrations? I downloaded 1.1.1h version from Release page. Now during matecat.sql import no errors are thrown but when i try to upload file to matecat, I get two kind of errors:
 One is Failed to store segments to database.
![image](https://cloud.githubusercontent.com/assets/20355306/22014399/224d68c6-dca6-11e6-892f-6f62e4b416bf.png)
Another one is Failed to create job. 
![image](https://cloud.githubusercontent.com/assets/20355306/22014379/10dcf552-dca6-11e6-82a7-2edce331d3a5.png)
MySQL GRANTS
![image](https://cloud.githubusercontent.com/assets/20355306/22015448/bc078aba-dcaa-11e6-8f93-66e540647599.png)
Files are created in storage
Access rigths are set to www-data:
![image](https://cloud.githubusercontent.com/assets/20355306/22015474/e44b1faa-dcaa-11e6-9852-797bf43fef4e.png)

Thanks

**Comment by wsuphappyhour** [comment]
Hi it seems that the cause of this was sql_mode='STRICT_TRANS_TABLES,NO_ENGINE_SUBSTITUTION'.
Out of the box install of MySQL 5.6.33 has STRICT_TRANS_TABLES,NO_ENGINE_SUBSTITUTION after changing it in /etc/mysql/my.cnf to sql_mode= error dissapears.
Thanks

**Comment by Ostico** [comment]
Yes exactly, this is a know configuration, i see now that we forgot to add this to the configuration guide.
Tnx @wsuphappyhour , we will fix.

**Comment by Ostico** [comment]
I suggest this configuration:
```
sql-mode="NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION"
```

**Comment by wsuphappyhour** [comment]
Thanks!
